src/utils/cookie.ts

```diff
@@ -12,6 +12,12 @@ export type CookieOptions = {
   signingSecret?: string
   sameSite?: 'Strict' | 'Lax' | 'None'
   partitioned?: boolean
+  securePrefix?: boolean
+  hostPrefix?: boolean
+}
+export type CookiePrefixOptions = {
+  secure?: boolean
+  host?: boolean
```

Comment:
I don't think there are cases where you want to use two prefixes, so I think it's better to design it so that you choose one or the other. That way, it's easier to understand that only one prefix is being used.

```suggestion
  prefix?: 'host' | 'secure'
}
export type CookiePrefixOptions = {
  prefix: 'host' | 'secure'
```

Reply:
Ah, this is a good suggestion. Thanks @Code-Hex !